src/index.tsx

```diff
@@ -35,8 +36,13 @@ if (window.ethereum) {
 }
 
 function Updaters() {
+  const location = useLocation()
```

Comment:
is there any issue with using a hook here?

Reply:
no, `<Updaters />` is a child of  `<Router>`. but let's switch just in case someone moves this code in the future

Reply:
sorry i spoke too soon ... when i tested without `useLocation` it breaks. because react-router handles navigations, window.location doesn't update the way we'd need it to